modules/claims_api/config/routes.rb

```diff
@@ -66,9 +66,10 @@
     namespace :veterans do
       get '/:veteranId/claims', to: 'claims#index'
       get '/:veteranId/claims/:id', to: 'claims#show'
-      get '/:veteranId/power-of-attorney', to: 'power_of_attorney#show'
-      put '/:veteranId/power-of-attorney:appointIndividual', to: 'power_of_attorney#appoint_individual'
-      get '/:veteranId/intent-to-files/:type',
+      get '/:veteranId/power_of_attorney', to: 'power_of_attorney#show'
+      put '/:veteranId/power_of_attorney/appoint_individual', to: 'power_of_attorney#appoint_individual'
+      put '/:veteranId/power_of_attorney/appoint_organization', to: 'power_of_attorney#appoint_organization'
```

Comment:
using the previous format of  `'/:veteranId/power-of-attorney:appointIndividual'` was fine on its own, but something about this (I'm guessing the colon) breaks when adding in `'/:veteranId/power-of-attorney:appointOrganization'`, as well. The result was both of these request URLs just getting mapped to `appointIndividual`. 

I switched it to underscores to be more consistent with Ruby conventions, but I'm not married to that. It seems like there's already inconsistencies across Lighthouse, between us and the health APIs using camel case, and appeals stuff using underscores, so I'm fine with whatever as long as we're internally consistent

Reply:
I don't have a problem switching to underscores.  Consistency within the app is the most important thing I think.  👍

Reply:
> Nothing in building of an api should be done because of the standard of the language its being implemented in

That was poorly worded. I assumed that things weren't working for some `{Rails Magic}` reason, which it seems was the constraints bit. The lighthouse convention, at least on the the `vets-api/modules` end, was underscores, so it seemed more consistent. Between that and underscores, I'm mostly indifferent. I do kinda lean towards 
```
/:veteranId/power-of-attorney/appoint-individual
/:veteranId/power-of-attorney:appoint-organization
```
just because that format seems more common and doesn't require the constraints. But this was mainly to get things working and not meant to be a larger naming question, so I'm happy to go with y'all's consensus.

Reply:
So this works as expected and doesn't require you to add the constraints argument?
```
/:veteranId/power-of-attorney/appoint-individual
/:veteranId/power-of-attorney:appoint-organization
```

If that's true, then I think this is fine.  I actually like the custom verb part not being snake-cased and instead being hyphenized like this.

Reply:
@jw81 I just realized I made a typo. second link would just be
```
/:veteranId/power-of-attorney/appoint-organization
```
but yeah, I think it reads more cleanly too, so I'm down to just keep it as such.

Reply:
Ah, hmmmmm.  I was thinking we were still going to use the custom verb syntax that Lee established.  I just thought you were going to switch the custom verb portion to hyphen-case instead of snakeCase.  If it's possible to keep the custom verb format and allowing the urls to continue to being hyphen-case, then I think we should do that.